projenrc/benchmark-test.ts

```diff
@@ -146,10 +143,13 @@ export class BenchmarkTest {
             },
           },
           {
-            name: 'Get TSC version',
-            id: 'get_tsc_version',
+            name: 'Get version line',
+            id: 'version',
             if: `github.event.repository.fork == false && github.ref == 'refs/heads/main'`,
-            run: `echo "TSC_VERSION=$(tsc --version | awk '{print $2}')" >> $GITHUB_ENV`,
+            run: [
+              `VERSION=$(node -p "require('./lib/version.js').RELEASE_LINE")`,
```

Comment:
I'm not sure if this will work in this spot, purely because the code might not be available here or build.

To test you can temporarily remove the step condition (`if`) and see if it works.

Reply:
Looks like it is not available: https://github.com/aws/jsii-compiler/actions/runs/9567139683/job/26374640771#step:4:13

where do you find this `version.js` file? @mrgrain ?

Reply:
The step just needs to happen as part of the `build` job, not the `benchmark` job

Reply:
Rewrote it to get the release line from the TSC version and [successfully tested](https://github.com/aws/jsii-compiler/actions/runs/9570795053/job/26386749518#step:4:9) it by removing the if statement on the step, so should be good to go now 👍